### ISSUE TITLE
Sample deploy hooks for chef?

### ISSUE THREAD

**Issue by zspencer** [engineyard/ey-docs#148]
Would be great if deploy hooks had an example of how to automatically run your chef scripts pre or post deploy.

**Comment by janeday** [comment]
Yes, the chef doc pages are scheduled for a rewrite. I've made a note of this request.
